sphinx/builders/latex.py

```diff
@@ -56,6 +57,22 @@ def init(self):
         self.docnames = []          # type: Iterable[unicode]
         self.document_data = []     # type: List[Tuple[unicode, unicode, unicode, unicode, unicode, bool]]  # NOQA
         self.usepackages = []       # type: List[unicode]
+        self.svg_support = False
+        with open(os.devnull, 'w') as FNULL:
+            try:
+                retcode = subprocess.call(['inkscape', '--version'],
+                                          stdout=FNULL,
+                                          stderr=subprocess.STDOUT)
+                if retcode == 0:
+                    self.supported_image_types.append('image/svg+xml')
+                    self.svg_support = True
+            except OSError:
+                pass
+        if not self.svg_support:
+            self.info("Latex builder initialized without svg support. You can"
```

Comment:
I would print this line only if the project actually contains SVG images (i.e. move it below).

Also, please capitalize “LaTeX” and “SVG” properly.